### PR TITLE
ticdc: update the doc link of filter rule

### PR DESCRIPTION
**Pull request by leoppro**
### What is changed, added or deleted? (Required)

update the doc link of filter rule

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

**Comment by leoppro** [comment]
@amyangfei PTAL

File: ticdc/manage-ticdc.md [hunk 1]
```diff
@@ -410,7 +410,7 @@ case-sensitive = true
 ignore-txn-start-ts = [1, 2]
 
 # 过滤器规则
-# 过滤规则语法：https://github.com/pingcap/tidb-tools/tree/master/pkg/table-filter#syntax
+# 过滤规则语法：https://docs.pingcap.com/zh/tidb/stable/table-filter
```
Comment (leoppro) [inline ticdc/manage-ticdc.md:413]:
```suggestion
# 过滤规则语法：https://docs.pingcap.com/zh/tidb/stable/table-filter#%E8%A1%A8%E5%BA%93%E8%BF%87%E6%BB%A4%E8%AF%AD%E6%B3%95
```

**Comment by leoppro** [comment]
@TomShawn  PTAL

File: ticdc/manage-ticdc.md [hunk 2]
```diff
@@ -410,7 +410,7 @@ case-sensitive = true
 ignore-txn-start-ts = [1, 2]
 
 # 过滤器规则
-# 过滤规则语法：https://github.com/pingcap/tidb-tools/tree/master/pkg/table-filter#syntax
+# 过滤规则语法：https://docs.pingcap.com/zh/tidb/stable/table-filter#%E8%A1%A8%E5%BA%93%E8%BF%87%E6%BB%A4%E8%AF%AD%E6%B3%95
```
Comment (TomShawn) [inline ticdc/manage-ticdc.md:413]: 建议不在注释里添加这么长的网址链接
```suggestion
# 过滤规则语法：https://docs.pingcap.com/zh/tidb/stable/table-filter#表库过滤语法
```

File: ticdc/manage-ticdc.md [hunk 3]
```diff
@@ -420,6 +420,7 @@ worker-num = 16
 [sink]
 # 对于 MQ 类的 Sink，可以通过 dispatchers 配置 event 分发器
 # 支持 default、ts、rowid、table 四种分发器
+# matcher 匹配语法和过滤器相同
```
Comment (TomShawn) [inline ticdc/manage-ticdc.md:423]:
```suggestion
# matcher 的匹配语法和过滤器规则语法相同
```

**Pull request merged**